sources/Context.cpp

```diff
@@ -236,15 +236,13 @@ Context::exportOutputJob() {
     return;
 
   switch (def_.simulationKind) {
-  case SimulationKind::SECURITY_ANALYSIS:
+  case dfl::inputs::Configuration::SimulationKind::SECURITY_ANALYSIS:
     // For security analysis always export the main jobs file, as dynawo-algorithms will need it
-    outputs::Job::exportJob(jobEntry_, absolute(def_.networkFilepath), config_.outputDir());
+    outputs::Job::exportJob(jobEntry_, absolute(def_.networkFilepath), config_.outputDir(), config_);
```

Comment:
I think there is no need to pass config_.outputDir() and config_, we can give simply config and get the outputDir value directly in exportJob method